core/modules/memman/MemManReal.cc

```diff
@@ -91,26 +92,20 @@ MemMan::Statistics MemManReal::getStatistics() {
 
 MemMan::Status MemManReal::getStatus(Handle handle) {
 
-    Status status;
-
     // First check if this is a valid handle and, if so, find it in our cache.
     // Once found, get its real status from the file set object.
     //
     if (handle != HandleType::INVALID && handle != HandleType::ISEMPTY) {
-       hanMutex.lock();
+       std::lock_guard<std::mutex> lg(hanMutex);
        auto it = hanCache.find(handle);
        if (it != hanCache.end() && it->second->isOwner(_memory)) {
-          status = it->second->status();
-          hanMutex.unlock();
-          return status;
+          return it->second->status();
        }
-       hanMutex.unlock();
      }
```

Comment:
Simply editorial -- lot's of braces for one line statements, eh?

Reply:
Perhaps, I like having them whenever the entire if statement doesn't fit on 1 line.